docs/core/entity/weather.md

```diff
@@ -23,6 +23,7 @@ Properties should always only return information from memory and not do I/O (lik
 | humidity | float | `None` | The current humidity in %.
 | ozone | float | `None` | The current ozone level.
 | cloud_coverage | int | `None` | The current cloud coverage in %.
+| uv_index | float | `None` | The current UV index.
```

Comment:
Can we document what definition of uv index, maybe with a link here https://en.wikipedia.org/wiki/Ultraviolet_index ?

Reply:
We can obviously provide the link but it seems most integrations has sort of their own "definition" so the integrations should probably clarify in their own respective documentation what UV index means.

Reply:
Let me take a look before we revert it so ensure what this means.

Reply:
Naively, my assumption is that we were talking about the int values from 
 https://www.who.int/news-room/questions-and-answers/item/radiation-the-ultraviolet-(uv)-index

Reply:
I must have confused this with something else. While this does not have a unit of measurement it's pretty clear what the numbers represent. 
Will update the doc once I'm back from vacation.

Reply:
Ok let me fix this:

```suggestion
| uv_index | float | `None` | The current [UV index](https://en.wikipedia.org/wiki/Ultraviolet_index).
```